src/shapes/Circle.java

```diff
@@ -0,0 +1,18 @@
+package shapes;
+
+public class Circle implements Shape {
+
+
+    public Circle(int i) {
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
```

Comment:
according to a school formula S = pi * r^2 :)
`return Math.PI * Math.square(radius)`